### ISSUE TITLE
Making code blocks/console blocks scrollable so that the code does not extend out of the block

### ISSUE THREAD

**Issue by soham96** [sci-eco/sci-eco.github.io#16]
Check out this image: 
<img width="1013" alt="Screenshot 2020-05-13 at 11 31 07 AM" src="https://user-images.githubusercontent.com/18757535/81768368-78adf000-950d-11ea-8d5c-38123b9dfafe.png">

**Comment by soham96** [comment]
So I was trying to figure out how to solve it and I did the following:

Added: `overflow-y: auto !important;`
Removed: `white-space: pre-wrap;`

Behaviour changes:
- Both type of blocks now scroll which is pretty cool
- No more wrapping behaviour. The wrapping caused the output of the model summary to be unreadable and the model training summary to wrap around and make it a bit tough to understand. Both of these do not wrap and it is more readable
- Since there is no more wrapping, reader will have to scroll significantly to read the model training summary. This can be solved by printing a shorter summary, but cannot be solved for the current quantization blog.

I also do not know if what I did is 'good practice' so I did not push it. Just hoping that it might help @prasannakumaar28 and help you get an idea of what the best way to make it scrollable is.
